frontend/src/components/Body/Tarifs/Tariffs.js

```diff
@@ -33,6 +41,12 @@ const StyledButton = styled(Button)`
   }
 ` 
 
+const StyledButtonContainer = styled(Box)`
+  display: flex;
+  justify-content: center;
+  margin-top: auto;
+`;
+
```

Comment:
Remove const. Move styles to a "sx" props. Leave a comment like "Container for.."